src/main/resources/application.yml

```diff
@@ -62,7 +57,9 @@ pebble:
   cache: false
   strict-variables: true
   suffix: '.peb'
-
+  
+server:
+  forward-headers-strategy: framework
```

Comment:
should use `native` unless we have a specific need, also, we already have `server` values at the top of the file, please combine them.

Reply:
combined.  I have no way of testing native because I don't know what the tomcat config is in production.